Sources/Core/Tracker/TrackerDefaults.swift

```diff
@@ -32,4 +32,5 @@ class TrackerDefaults {
     private(set) static var platformContext = true
     private(set) static var geoLocationContext = false
     private(set) static var screenEngagementAutotracking = true
+    private(set) static var immersiveSpaceContext = false
```

Comment:
We don't know how many people will use the visionOS features, but likely not the majority of users. Seems strange to me to have this on even if the overhead is minimal, when e.g. lifecycle tracking is off by default even though that's something that everyone might find useful.

Reply:
I want to enable lifecycle tracking by default for v6 (#852) because it's necessary for screen engagement tracking which will also be on by default.

Up to you, but I don't really see a good reason not to enable it by default – it just makes it more of an effort and confusing for users (admittedly a very small minority of users) who do want to track these events to do it.

Reply:
In that case, entities for all!